js/PageLevelProgressIndicatorView.js

```diff
@@ -51,6 +52,12 @@ class PageLevelProgressIndicatorView extends Backbone.View {
   }
 
   calculatePercentage() {
+    const isPrensentationComponentWithItems = (!this.model.isTypeGroup('question') && this.model instanceof ItemsComponentModel);
+    if (isPrensentationComponentWithItems) {
```

Comment:
Should be `isPresentationComponentWithItems` not `isPrensentationComponentWithItems`.